rosdep/base.yaml

```
@@ -2196,6 +2196,9 @@ imagemagick:
   gentoo: [virtual/imagemagick-tools]
   nixos: [imagemagick]
   ubuntu: [imagemagick]
+intel-opencl-icd:
+  debian: [intel-opencl-icd]
+  ubuntu: [intel-opencl-icd]
```

This package is not available in Ubuntu Bionic so the distribution version cannot be elided.

```suggestion
  ubuntu:
    '*': [intel-opencl-icd]
    bionic: null
```

---

Hadn't checked that thoroughly, thanks for the fix